hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/client/HttpFSFileSystem.java

```diff
@@ -1743,6 +1744,19 @@ public BlockLocation[] getFileBlockLocations(final FileStatus status,
     return getFileBlockLocations(status.getPath(), offset, length);
   }
 
+  @Override
+  public FileStatus getFileLinkStatus(final Path f) throws IOException {
+    Map<String, String> params = new HashMap<>();
+    params.put(OP_PARAM, Operation.GETFILELINKSTATUS.toString());
+    HttpURLConnection conn =
+        getConnection(Operation.GETFILELINKSTATUS.getMethod(),
+            params, f, true);
```

Comment:
one line may be batter

```
HttpURLConnection conn =
        getConnection(Operation.GETFILELINKSTATUS.getMethod(), params, f, true);
```

Reply:
Thanks for your suggestion, I'll make the required code changes promptly